presto-main/src/main/java/com/facebook/presto/operator/aggregation/ApproximateAverageAggregations.java

```diff
@@ -79,15 +80,15 @@ public static void combine(ApproximateAverageState state, ApproximateAverageStat
         }
     }
 
-    @OutputFunction(StandardTypes.VARCHAR)
+    @OutputFunction(VARCHAR_MAX_LENGTH)
```

Comment:
Can you split this commit into one that does the refactoring to enable this, and then a second commit to change all the functions to `varchar(...)`?